src/gui/org/deidentifier/arx/gui/view/impl/utility/ViewClassificationAttributes.java

```diff
@@ -233,9 +216,19 @@ private void update() {
         
         for (int col = 0; col < handle.getNumColumns(); col++) {
             String attribute = handle.getAttributeName(col);
-            Image image = controller.getResources().getImage(model.getInputDefinition().getAttributeType(attribute));
-            createTableItem(attribute, image, features, model.getSelectedFeatures());
-            createTableItem(attribute, image, classes, model.getSelectedClasses());
+            DataDefinition def = model.getOutputDefinition() == null ? model.getInputDefinition() : model.getInputDefinition();

```

Comment:
This line contains a bug. It should read:
DataDefinition def = model.getOutputDefinition() == null ? model.getInputDefinition() : model.getOutputDefinition();